# Buttons
uint8 SOFT_BUTTON = 0
uint8 UP_BUTTON = 1
uint8 START_BUTTON = 2
uint8 BACK_BUTTON = 3
uint8 DOWN_BUTTON = 4

uint8   button         #values defined above
bool    engaged        #whether or not the button is down or not